measures/measures-schema.yaml

```diff
@@ -110,6 +110,14 @@ definitions:
         description: Possible methods for submitting performance data for the measure.
         type: array
         items: { $ref: #/definitions/methods }
+      eligibilityOptions:
+        description: Eligibility options mirror denominator options in measure specifications. Each option comprises a set of codes used to identify eligible instances of the associated measure.
```

Comment:
Not sure I understand - I thought these specifications were also for claims to quality processing of claims, not sure how QCDRs factor in, could you expand?